package.json

```diff
@@ -14,7 +14,8 @@
   },
   "scripts": {
     "test": "xo && mocha",
-    "bench": "matcha bench.js"
+    "bench": "matcha bench.js",
+    "jshint": "jshint index.js"
```

Comment:
@shinnn I think you must've meant to link to a CI job that wasn't related to my PR.

If `xo` isn't installed globally and it isn't being installed in the local `node_modules` directory, why is the command executing?